cclib/method/moments.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Calculation of electric multipole moments based on data parsed by cclib."""
+
+import numpy
+
+from cclib.parser.utils import convertor
+from cclib.method.calculationmethod import Method
+
+
+class Moments(Method):
+    """This class calculates multipole moments and stores them in
```

Comment:
Could you change this into a one-line summary and a separate paragraph for everything else? Please see PEP 257 (https://www.python.org/dev/peps/pep-0257/) for generally accepted docstring style conventions.